src/core/indexedlist.h

```diff
@@ -13,30 +13,91 @@ class IndexedList
     explicit IndexedList() = default;
```

Comment:
Thanks for taking a look inside. No idea why this exists. I just didn't want to change "too much". Though it looks like this IndexedList is used solely for management of ToxCall objects. Btw.: a QMap/QHash would be the appropriate thing to replace it - it is actually about key-value mapping.